### ISSUE TITLE
Editor with defaultValue always displays with focus set

### ISSUE THREAD

**Issue by spallister** [niuware/mui-rte#158]
## Checklist

- [x] This is not covered in the [repository examples](https://github.com/niuware/mui-rte/tree/master/examples).
- [x] I am using the package [latest version](https://github.com/niuware/mui-rte/releases/latest).
- [x] This is a bug report or an enhancement proposal.

## Description

First, thanks for a great library!

Upon setting a `defaultValue`, for me the editor always loads with focus set.  Which makes sense from looking at `MUIRichTextEditor.tsx`, where if the editor is not in `readOnly` mode, editor state is set as `setEditorState(EditorState.moveFocusToEnd(nextEditorState))` (line 314).  

Because I would like to display the editor without setting focus, I was hoping that I could simply change this to `setEditorState(EditorState.moveSelectionToEnd(nextEditorState))` (i.e. replace `moveFocusToEnd` with `moveSelectionToEnd`), after reading through the Draft.js documentation.  Unfortunately, I'm seeing that focus is still being set upon making this change (i.e. I set a default value for the basic example).  Do you happen to know why this may be happening?  Is focus being set elsewhere by mui-rte?

**Comment by niuware** [comment]
I believe currently there is no other place where the focus is set. What is your use case? Have you tried focusing other HTML element after loading your page?

**Comment by longsangstan** [comment]
i am having the same issue... when the default value is set, the editor is being focused automatically when it loads

**Comment by acabreragnz** [comment]
I'm having the same issue and I'm not sure whether this always has being working in this way, but when you have a `defaultValue` the textarea is auto focused.

```json
"mui-rte": "1.24.0",
```

This my workaround using hooks and ref:

```javascript
  const inputToFocusRef = useRef(null);

  useEffect(() => {
    const timeout = setTimeout(() => {
      // We are manually setting the focus because the MUIRichTextEditor component
      // has autofocus when it has a defaultValue
      if (inputToFocusRef && inputToFocusRef.current) {
        inputToFocusRef.current.focus();
      }
    });

    return () => {
      clearTimeout(timeout);
    };
  }, []);
```
```html
  <!-- this is another input different thant the mui-rte -->
   <input
       ref={inputToFocusRef}
       ...
  />
```

**Comment by imryanjay** [comment]
Will this be addressed anytime soon?  When page loads, focus is set on the last instance of <MUIRichTextEditor /> with a defaultValue prop passed.  Huge breaking change for a form.

**Comment by alanmortoni** [comment]
In a form, if start typing on mui-rte then I type in another text field, it keeps stealing the focus from my text field to mui-rte.

Any fix for this?

**Comment by niuware** [comment]
This is fixed on version `1.26.0`.
